### DISCUSSION TITLE
How to get types payload instance

### DISCUSSION THREAD

**Discussion by teunmooij** [payloadcms/payload#2498] [Q&A]
What's the best way to get a typed payload instance?
In the live video's I see import from `../payload`, but that fule is never shown and I can't find it in any of the example repos.
In my own project I'm currently exporting it from a central file typed as `BasePayload<Config>`, but `BasePayload` is not officially exposed from the package, so it feels a bit wrong.

**Comment by itsjxck** [comment]
A fully typed Payload instance would be great, it seems like a solution similar to how [Prisma solves a similar problem](https://www.prisma.io/docs/concepts/components/prisma-client) could be beneficial.

**Reply by teunmooij** [reply]
Payload is already typed. The only problem is that the export I found is in the `/dist`. So it's not on available though the `payload`, `payload/types` or `payload/config` imports. Hence my question.
